voicebox_pytorch/voicebox_pytorch.py

```diff
@@ -28,6 +29,7 @@
 
 from vocos import Vocos
 
+_LOGGER = logging.getLogger(__file__)
```

Comment:
yea looks good, want to just remove that underscore?  `LOGGER` is fine, or even `debug_print = logging.getLogger(__file__).debug`

Reply:
OK